sdk/src/azure/core/az_base64.c

```diff
@@ -370,17 +376,40 @@ AZ_NODISCARD int32_t az_base64_get_max_encoded_size(int32_t source_bytes_size)
   return (((source_bytes_size + 2) / 3) * 4);
 }
 
-static AZ_NODISCARD int32_t _az_base64_decode(uint8_t* encoded_bytes)
+static int32_t _get_base64_decoded_char(int32_t index, _az_base64_mode mode)
+{
+  if (mode == _az_base64_mode_url)
+  {
+    if (index == 43 || index == 47)
```

Comment:
Magic numbers like 43, 45, or 95 confuse me. Could you use '-'/'_'/'+'/'/' instead of their ASCII equivalents? Or #define a value for the numeric constants?

Reply:
Ahhh yes absolutely